src/ingestors/fxhash/abi.ts

```diff
@@ -0,0 +1,627 @@
+  {
+    anonymous: false,
+    inputs: [{ indexed: true, internalType: 'address', name: 'pendingOwner', type: 'address' }],
+    name: 'OwnershipHandoverRequested',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: 'oldOwner', type: 'address' },
+      { indexed: true, internalType: 'address', name: 'newOwner', type: 'address' },
+    ],
+    name: 'OwnershipTransferred',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: '_token', type: 'address' },
+      { indexed: true, internalType: 'uint256', name: '_reserveId', type: 'uint256' },
+      { indexed: true, internalType: 'address', name: '_claimer', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: '_index', type: 'uint256' },
+    ],
+    name: 'PassClaimed',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [{ indexed: false, internalType: 'address', name: 'account', type: 'address' }],
+    name: 'Paused',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: '_token', type: 'address' },
+      { indexed: true, internalType: 'uint256', name: '_reserveId', type: 'uint256' },
+      { indexed: true, internalType: 'address', name: '_buyer', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: '_amount', type: 'uint256' },
+      { indexed: false, internalType: 'address', name: '_to', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: '_price', type: 'uint256' },
+    ],
+    name: 'Purchase',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: '_token', type: 'address' },
+      { indexed: true, internalType: 'uint256', name: '_reserveId', type: 'uint256' },
+      { indexed: true, internalType: 'address', name: '_claimer', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: '_index', type: 'uint256' },
+    ],
+    name: 'SlotClaimed',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [{ indexed: false, internalType: 'address', name: 'account', type: 'address' }],
+    name: 'Unpaused',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: '_token', type: 'address' },
+      { indexed: true, internalType: 'address', name: '_creator', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: '_proceeds', type: 'uint256' },
+    ],
+    name: 'Withdrawn',
+    type: 'event',
+  },
+  {
+    inputs: [
+      { internalType: 'address', name: '_token', type: 'address' },
+      { internalType: 'uint256', name: '_reserveId', type: 'uint256' },
+      { internalType: 'uint256', name: '_amount', type: 'uint256' },
+      { internalType: 'address', name: '_to', type: 'address' },
+    ],
+    name: 'buy',
```

Comment:
For sanity of diff sizes, we favor trimmed ABIs — i.e. including only the objects describing the methods we care about.

In this case, everything except the `buy` and `prices` methods can be deleted.

Reply:
I've updated the README based on this too 🙏 
https://github.com/floornfts/mobile-minting/commit/ae8208eeb18d27a25b012e7f8b465d17b26829de